Source/Launcher/DualityLauncher.csproj

```diff
@@ -8,24 +8,25 @@
 		<RootNamespace>Duality.Launcher</RootNamespace>
 		<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
 		<ApplicationIcon>DualityIcon.ico</ApplicationIcon>
-		<OutputType>Exe</OutputType>
+		<OutputType>WinExe</OutputType>
 		<Version>4.0.0-alpha3</Version>
+		<Packable>false</Packable>
 	</PropertyGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\Core\Primitives\DualityPrimitives.csproj">
-      <Project>{c87a6a2f-e537-48fa-a789-3c54cf29cc09}</Project>
-      <Name>DualityPrimitives</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Core\Duality\Duality.csproj">
-      <Project>{D821AFB9-BCC9-4025-9A9C-798E2BC546AB}</Project>
-      <Name>Duality</Name>
-    </ProjectReference>
-  </ItemGroup>
+	<ItemGroup>
```

Comment:
Seems it now uses tabs for indenting even though the editor config should set it to spaces for csproj files..